src/theme/components/card.ts

```diff
@@ -128,6 +128,8 @@ const CardActions = {
 const CardContent = {
   baseStyle: {
     marginBottom: "xs",
+    width: "100%",
```

Comment:
I can’t seem to recreate a scenario in Storybook where I see these styles applied.  Where is this coming into. play?  Also, did you see the last comment from Crystal on this in the issue tracker, about not needing to implement the style (yet)?